deployments/pulse3d/services/pulse3d/src/models/models.py

```diff
@@ -18,15 +18,15 @@ class UploadResponse(BaseModel):
 
 class JobRequest(BaseModel):
     upload_id: uuid.UUID
-
+    baseline_widths_to_use: Optional[TupleParam]
```

Comment:
It would be good to add a test for this param. `test_jobs__post__advanced_params_given` is similar to what a test for this param will look like, but you'll have to make a new test for it since it is different enough